### ISSUE TITLE
SUPPORT TICKET

### ISSUE THREAD

**Issue by viviancan** [Iridescent-CM/technovation-app#4325]

**Comment by viviancan** [comment]
We received[ feedback from a US-based ](https://iridescentlearning.slack.com/archives/C06CHBQ4E/p1699991694089119)mentor stating they could not complete their mentor background check. After further investigation, it appears that this individual, while US geographically based, did not have an SSN, which Checkr requires to process a US background check. VET is moving forward with the volunteer in a different volunteer role, however, we may need to revisit this type of scenario in the future.

**Comment by dboyer** [comment]
We received feedback from a ChA that their profile was incorrectly updated with data connected to a mentor. Details in [this slack message](https://iridescentlearning.slack.com/archives/C06CHBQ4E/p1700601078996399). Tagging @viviancan to add more details as she looks into it.

**Comment by shaun-technovation** [comment]
> We received feedback from a ChA that their profile was incorrectly updated with data connected to a mentor. Details in [this slack message](https://iridescentlearning.slack.com/archives/C06CHBQ4E/p1700601078996399). Tagging @viviancan to add more details as she looks into it.

Just following up with this from a security perspective. From what I can tell, there isn't a security vulnerability that is allowing this.

A mentor or chapter ambassador's account gets updated via:
https://github.com/Iridescent-CM/technovation-app/blob/be58e19d0102ece1d1cfdd3d0e9fb21325addad3/app/technovation/profile_updating.rb#L12

which needs a `profile` to update. In a mentor's case, it's the `current_profile` which will be the current mentor logged in or the one being impersonated:
https://github.com/Iridescent-CM/technovation-app/blob/be58e19d0102ece1d1cfdd3d0e9fb21325addad3/app/controllers/mentor_controller.rb#L56-L64

For a chapter ambassador it will also be the `current_profile` which will also be the current chapter ambassador logged in or the one being impersonated:
https://github.com/Iridescent-CM/technovation-app/blob/be58e19d0102ece1d1cfdd3d0e9fb21325addad3/app/controllers/chapter_ambassador_controller.rb#L29-L36

We aren't looking up someone by an ID that could be manipulated, so I'm pretty confident it's not a security vulnerability or loophole, if it was we would most likely be seeing more instances of this happening.

There could still be a loophole somewhere, but I'm not seeing it. 🤔 And to your point/comment, Deb, about more tracking/logging, well, I think that would be the most helpful in this situation because we would know who updated what and when.

**Comment by dboyer** [comment]
Users from Chile are reporting an issue with the error message when they try to upload too large of a file for the parental consent. It looks like the error text is not providing the correct max file size.

A video of the issue is [saved here](https://drive.google.com/file/d/16oKnLJhb5M_MxHM6vGx6RzEaADCt0g-w/view?usp=sharing).

**Comment by dboyer** [comment]
Additional location issue reported where the Chapter Ambassador for [Lagos](/participants/178535#/) is appearing for participants in Ogun State rather than the ChA from [Ogun](/participants/178678#/). This is true even when I logged in as a mentor who had identified her location as the same city associated with the ChA from Ogun.

More info in the [Slack thread here](https://iridescentlearning.slack.com/archives/C06CHBQ4E/p1700589561832089).

I'm betting this is the same problem we saw in Pakistan with chapters that are close together. Is there a way to override the default chapter in the code? Basically show the nearest chapter no matter what?

**Comment by shaun-technovation** [comment]
> Users from Chile are reporting an issue with the error message when they try to upload too large of a file for the parental consent. It looks like the error text is not providing the correct max file size.

I can fix the error text.

From the video, it looks like they're trying to upload a 4.9 MB file, the limit is 3 MB. We can bump that limit up of course, but for the time being until the error text gets updated and/or we increase the max file size, to get them unblocked, we can just tell them that the limit is 3 MB and to upload files less than that size.

**Comment by shaun-technovation** [comment]
> > Users from Chile are reporting an issue with the error message when they try to upload too large of a file for the parental consent. It looks like the error text is not providing the correct max file size.
> 
> I can fix the error text.
> 
> From the video, it looks like they're trying to upload a 4.9 MB file, the limit is 3 MB. We can bump that limit up of course, but for the time being until the error text gets updated and/or we increase the max file size, to get them unblocked, we can just tell them that the limit is 3 MB and to upload files less than that size.

This issue should be fixed on QA now. This is the error message that gets displayed when **_both_** consent forms are over the max file size limit.

I also created a separate issue for this: #4368, so we can track its progress.

**Comment by dboyer** [comment]
Remove sidebar_b - split off from #4297
